simpledotorg/simple-server
# Add district reports template

**Story card:** No story

## Because

Mahima's new feature needs a link to the blank Excel template for CVHOs.

## This addresses

Launch of Excel Reports

## Test instructions

Click link under Settings > Resources.
Make sure you can download the Excel.
Should only display in India.
